Makefile

```diff
@@ -8,40 +6,38 @@ else
 endif
 
 TARGETS=awesome
-BUILDLN=build
+BUILDDIR=build
 
-all: $(TARGETS) $(BUILDLN) ;
+all: $(TARGETS) ;
 
 $(TARGETS): cmake-build
-	ln -s -f ${builddir}/$@ $@
-
-$(BUILDLN):
-	test -e $(BUILDLN) || ln -s -f ${builddir} $(BUILDLN)
+	ln -s -f $(BUILDDIR)/$@ $@
```

Comment:
Why doesn't this make sense? This is the rule for `./awesome` and it creates `./awesome`.